webapp/scripts/functions.js

```diff
@@ -100,7 +100,8 @@ module.exports = {
    */
   getUnivarBounds(mapset, map, column) {
     const univar = grass(mapset, `v.db.univar -e -g map=${map} column=${column}`).trim().split('\n')
-    return [univar[1].split('=')[1], univar[2].split('=')[1]]
+    const [min, max] = [univar[1].split('=')[1], univar[2].split('=')[1]]
```

Comment:
`univar.slice(1, 2).map(v => v.split('=')[1])`

Reply:
aha! nice

Reply:
oops, should be: `univar.slice(1, 3).map(v => v.split('=')[1])` :)